src/js/controller/controller.js

```diff
@@ -142,10 +142,8 @@ Object.assign(Controller.prototype, {
         _this.on(AD_PAUSE, event => changeReason(null, event.pauseReason));
 
         _model.on('change:scrubbing', function(model, state) {
-            if (state) {
+            if (state && model.get('state') !== STATE_PLAYING) {
```

Comment:
i think we can now access state with `model.state` thanks to rob's simple-model attribute changes.

Reply:
maybe... almost.

Reply:
Why not always call pause? Checking the state shouldn't be necessary. Don't add extra code that we don't need. There shouldn't be any harm in calling pause when paused.

Reply:
I was checking ```model.get('state')``` because we don't want to call pause when seeking while the video is still playing.

Reply:
The tag needs to be paused while scrubbing or you will hear audio and see playback jitteres as you scrub (drag the slider).

Reply:
Still waiting on this feedback to be addressed